pypeit_test

```diff
@@ -108,9 +112,42 @@ def main():
         # Setups
         setups = next(os.walk(os.path.join(raw_data, instr)))[1]
 
+        # Cut me down
         if pargs.debug and ('blue' not in instr):
             continue
 
+        # Develop cut
+        if pargs.tests == 'develop':
+            if instr == 'Keck_DEIMOS':
+                req_setups = ['600ZD_M_600', '1200G_M_7750', '830G_LVM_8400', '830G_M_8500', '830G_L_8100']
+            elif instr == 'Gemini_GNIRS':
+                req_setups = ['GNIRS']
+            elif instr == 'Shane_Kast_blue':
+                req_setups = ['452_3306_d57', '600_4310_d55']
+            elif instr == 'Shane_Kast_red':
+                req_setups = ['600_7500_d55']
+            elif instr == 'Keck_NIRES':
+                req_setups = ['NIRES']
+            elif instr == 'Keck_LRIS_blue':
+                req_setups = ['long_400_3400_d560', 'long_600_4000_d560', 'multi_300_5000_d680', 'multi_600_4000_d560', 'multi_1200_340_d460']
+            elif instr == 'Keck_LRIS_red':
+                req_setups = ['long_600_7500_d560', 'multi_1200_9000_d680_1x2', 'multi_600_5000_d560',
+                              'multi_1200_9000_d680', 'multi_400_8500_d560']
+            elif instr == 'VLT_XSHOOTER':
```

Comment:
For X-shooter the UVB is not yet supported as we don't have a wavelength solution. All of the rest of the setups need to be tested, not this restricted list of binnings. X-shooter is the only instrument we have that tests cross-binning issues and this test uncovered many problems in processimages. So please remove UVB_1x1 make the list of tests here 

NIR VIS_1x1 VIS_2x1 VIS_2x2